src/plugins/nuxt/playbackProfiles/helpers/webmVideoFormats.ts

```diff
@@ -0,0 +1,28 @@
+import { Context } from '@nuxt/types';
+import { hasAv1Support, hasVp8Support, hasVp9Support } from './mp4VideoFormats';
+
+/**
+ * @param {Context} context - Nuxt context
+ * @param {HTMLVideoElement} videoTestElement - A HTML video element for testing codecs
+ * @returns {string[]} An array of supported codecs
+ */
+export function getSupportedWebMVideoCodecs(
+  context: Context,
+  videoTestElement: HTMLVideoElement
+): string[] {
```

Comment:
Not really a fan of moving all the plugins into subdirectories.

Reply:
Why?

Reply:
After comparing it with master I can see that this is a net benefit - the nuxt folder just feels like a catch-all even though there's not really a better way to categorize them. It also looks like `directives` wasn't removed from its old location on your branch by the way.

Reply:
Good catch! Will fix it!